src/route/index.js

```
@@ -1,9 +1,12 @@
 const express = require("express");
 //const router = express.Router();
 //const login = require("../controller/login");
-const cluster = require("../controller/cluster");
+//const cluster = require("../controller/cluster");
 const registration = require("../controller/registration");
+const unusedVar = "";
 const userController = require("../controller/userController");
+const userController = require("../controller/userController");
```

Biome (1.9.4)

[error] Shouldn't redeclare 'userController'. Consider to delete it or rename it.

'userController' is defined here:


(lint/suspicious/noRedeclare)